tools/wptserve/setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 PACKAGE_VERSION = '3.0'
-deps = ["six>=1.13.0", "h2>=3.0.1"]
```

Comment:
I'm not sure if any setup.py files are actually used, but it seemed best to make this change. Do you know, @jgraham?

Reply:
OK, so I'll leave it in.